_posts/2018-07-04-aerogear-mobile-services-announcement.md

```diff
@@ -2,7 +2,9 @@
 layout: news
 section: news
 title: Announcing AeroGear Mobile Services on OpenShift
-author: aerogear
+author:
```

Comment:
@danielpassos I don't think you need to add the author meta data here. It should pick it up from the _data/people here: https://github.com/aerogear/aerogear.org/blob/master/_data/people.yml#L27

Reply:
Done!